### DISCUSSION TITLE
Only Create New Tabs When a URL is Loaded

### DISCUSSION THREAD

**Discussion by lugenx** [zen-browser/desktop#2588] [Ideas]
Pressing `Cmd+T` immediately creates a new blank tab, even if no URL is entered. This results in empty tabs accumulating in the tab bar, leading to workspace clutter, especially when tabs are opened by habit but left unused.

**Expected Behavior:**
Adopt behavior similar to Arc Browser, where pressing `Cmd+T` opens a URL entry field, but no new tab is created unless a URL is entered and loaded. If the user exits the URL entry without typing anything, no tab should be added to the tab list.

**Benefits:**
This change would prevent empty tabs from building up, helping users maintain a cleaner, more organized workspace with only active tabs displayed.

**Comment by liujip0** [comment]
Or, alternatively, a New Tab can be opened for users who like to see weather, pocket recommendations, etc, but no additional tabs will be opened when the user is already in a New Tab
